src/libmongoc/CMakeLists.txt

```diff
@@ -807,6 +807,7 @@ set_target_properties (mongoc_shared PROPERTIES
    SOVERSION 0
    pkg_config_REQUIRES "libbson-1.0"
    )
+mongo_generate_pkg_config(mongoc_shared INSTALL RENAME libmongoc-${MONGOC_API_VERSION}.pc)
```

Comment:
Consider using `FILENAME` for consistency with other `mongo_generate_pkg_config` calls.

Is there a use case for setting different values for `RENAME` and `FILENAME`? Consider removing `RENAME`.

Reply:
`RENAME` comes from `install(FILES ...)`. The split in this case helps with multi-config generators. We want to allow multi-conf generation to work in the build, but we only need to install one version at a time.